list1/src/Tests.java

```diff
@@ -0,0 +1,45 @@
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+
+public class Tests {
+    private Random r = new Random();
+    private int size = 1000;
```

Comment:
podpisuj dokładniej zmienne, size ale size of what? Tu jest prosta klasa i jeszcze się nie pogubisz, ale jak będziesz miał więcej zmiennych i stałych to można zwariować z takimi nazwami